test/integration/cart_integration_test.rb

```diff
@@ -8,14 +8,14 @@ class CartIntegrationTest < ActionDispatch::IntegrationTest
 
     click_link_or_button('Cart')
 
-    within('#cart_items') do
+    within('#cart_projects') do
       assert page.has_content?('Your Cart Is Empty')
     end
   end
 
-  test 'an unauthorized user can add an item to the cart' do
-    item = Item.create(title: 'coffee', description: 'black nectar of the gods', price: 1200)
-    visit "/items/#{item.id}"
+  test 'an unauthorized user can add an project to the cart' do
+    project = Project.create(title: 'coffee', description: 'black nectar of the gods', price: 1200)
```

Comment:
Line is too long. [99/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.